src/formio.wizard.js

```diff
@@ -310,8 +310,14 @@ export default class FormioWizard extends FormioForm {
 
       if (clickable) {
         this.addEventListener(pageButton, 'click', (event) => {
-          event.preventDefault();
-          this.setPage(i);
+          // Validate the form builed, before go to the next page
+          if (this.checkValidity(this.submission.data, true)) {
+            event.preventDefault();
+            this.setPage(i);
+          }
+          else {
```

Comment:
nit. format `} else {` in the same line

Reply:
I thought that but linter bothers me with: `Closing curly brace appears on the same line as the subsequent block. (brace-style)` 🙀